.github/workflows/wheels.yml

```diff
@@ -86,6 +90,12 @@ jobs:
             python: 312
             platform_id: manylinux_x86_64
             manylinux_image: manylinux2014
+          - os: ubuntu-latest
+            python: 313
+            platform_id: manylinux_x86_64
+            manylinux_image: manylinux2014
+            # TODO: remove next line when Python 3.13 is released
+            prerelease_pythons: True
```

Comment:
```suggestion
```

With cibuildwheel v2.20.0, the prerelease_python flag is not required anymore: https://cibuildwheel.pypa.io/en/stable/changelog/#v2200

>  CPython 3.13 wheels are now built by default - without the CIBW_PRERELEASE_PYTHONS flag.

(We'll still need to keep windows because of the `-rc` logic when building the image)